### DISCUSSION TITLE
Ordinary rolls

### DISCUSSION THREAD

**Discussion by matthias-ronge** [kitodo/kitodo-production#5636] [General]
Production comes with a collection of rolls. How about a description of the roles in natural language? What would a person, who was applied a specific role, need to be able to do? And what shouldn't she be able to do? I have the impression, that the rolls, given in the standard database, do not serve all purposes in a meaningful way. For a thorough configuration of the rolls, a description of their meanings is required. What is a _configure columns_? What is the difference between an _imaging_ and a _scanning_? Etc …

**Roll database:**
- Administration
- Client management
- Configure columns
- Database statistics
- Imaging
- Metadata
- Process management
- Projectmanagement
- Quality control
- Role management
- Scanning
- User management
- Workflow management

**Comment by andre-hohmann** [comment]
Thanks for the initiative. The documentation of roles would be helpful, as the message in the community mailinglist from 2023-04-14 shows. 

In my opinion, the [authorithies](https://github.com/kitodo/kitodo-production/wiki/Berechtigungen) should also be described. Everyone is welcomed to complete the list. 
For example, _configure columns_ consists only of the authority _Configure columns_ and in the SLUB Dresden the authority is assigned to all roles directly (Metadaten, Scanner, ...) instead of creating a new role.

In addition, the roles gather the authorities, which belong together and should be seen only as opportunities. In the SLUB Dresden, we have defined own [roles](https://github.com/kitodo/kitodo-production/wiki/SLUB-Dresden-Rollen), based on the available roles. Without these roles, it is difficult to start working in Kitodo.Production. 

I ask myself, if it would more helpful, if the institutions describe their roles, as for example. Advantages could be: 
- the roles are practically applied
- the concepts in the institutions can be compared
